--- origsrc/gnutls-3.3.24/lib/x509/output.c	2016-04-09 06:32:40.000000000 -0500
+++ src/gnutls-3.3.24/lib/x509/output.c	2016-08-28 21:41:17.557139100 -0500
@@ -23,6 +23,10 @@
 /* Functions for printing X.509 Certificate structures
  */
 
+#ifdef _WIN32
+# define _WIN32_WINNT 0x0600
+# include <ws2tcpip.h>
+#endif
 #include <gnutls_int.h>
 #include <common.h>
 #include <gnutls_x509.h>
@@ -61,7 +65,7 @@ char *ip_to_string(void *_ip, int ip_siz
 		return NULL;
 	}
 
-#ifdef HAVE_INET_NTOP
+#if defined HAVE_INET_NTOP || defined _WIN32
 	if (ip_size == 4)
 		return inet_ntop(AF_INET, _ip, string, string_size);
 	else
